src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImpl.java

```diff
@@ -106,6 +114,61 @@ public ProfessionalUser findProfessionalUserByEmailAddress(String email) {
         return professionalUserRepository.findByEmailAddress(RefDataUtil.removeAllSpaces(email));
     }
 
+    @Override
+    public ResponseEntity<Object> fetchUsersForRefresh(String since, String userId, Integer page, Integer size) {
+        ResponseEntity<Object> responseEntity;
+
+        if (since != null && page != null && size != null) {

```

Comment:
if (since != null && page != null && size != null) {
            Pageable pageable =
                    createPageableObject(page - 1, size, Sort.by(Sort.DEFAULT_DIRECTION, NESTED_ORG_IDENTIFIER)
                            .and(Sort.by(Sort.DEFAULT_DIRECTION, USER_IDENTIFIER)));
            return findRefreshUsers(since, pageable);
        } 
        return findSingleRefreshUser(userId);

Reply:
updated in commit [eb360dc](https://github.com/hmcts/rd-professional-api/pull/1529/commits/eb360dc1e9f8405d50ffea39011aa6a4e03c82a1)